example-runner/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=uk.gov.gchq.palisade.example.runner.config.ExampleClientConfiguration
```

Comment:
Just a question.  Here we are setting-up a starter for an AutoConfiguration so that means that there is a need for this to load the auto-configuration class (ExampleClientConfiguration) under specific/special conditions.  What are these conditions?  If is more than a sound bite perhaps a mention in the Java doc for the ExampleClientConfiguration class.

Reply:
I believe this was intended for the spring bits to work in the performance module when the example-runner is used as a mvn dependency.
Not sure where to comment upon it though